tools/generator/src/Extensions/XCScheme+Extensions.swift

```diff
@@ -240,6 +240,9 @@ extension XCScheme.LaunchAction {
             askForAppToLaunch: launchActionInfo.askForAppToLaunch ? true : nil,
             customWorkingDirectory: launchActionInfo.workingDirectory,
             useCustomWorkingDirectory: launchActionInfo.workingDirectory != nil,
+            enableAddressSanitizer: launchActionInfo.diagnostics?.enableAddressSanitizer ?? false,
```

Comment:
Even though this API seems to indicate you can have multiple enabled at once, it's really a radio button, right? If so, I think it should be more like `sanitizer = "address"` in the BUILD API and translate that (optional) enum into bools here?

Reply:
It's a checkbox but some of the functionality behaves like a radio button. (For ex - asan cannot be enabled with tsan). Eventually the diagnostics will be extended by other options such as: "Malloc Scribble", "Zombie Objects" etc, so a couple of them can be enabled at a time. 

I am thinking about this point - `sanitizer = "address" in the BUILD API `...

Reply:
So usan can be enabled with asan? If so, this makes sense, and we just need to error when they can't be enabled together.

I still like a `sanitizers` argument, with simple `address = True, thread = False` arguments under it. That way it's grouped like the UI in Xcode.

Reply:
Yes, asan/tsan can be enabled with ubsan.

>  I still like a sanitizers argument, with simple address = True, thread = False arguments under it. That way it's grouped like the UI in Xcode.

Yes, I agree, that would be a better representation. I will update the PR, thanks.